configure.py

```diff
@@ -463,6 +724,12 @@ def _fribidi_install(cfg):
 @_block(
     "nopegl-setup",
     {
+        "Android": [
+            _nopemd_install,
+            _glslang_install,
+            _harfbuzz_install,
+            _fribidi_install,
```

Comment:
missing freetype?

Adding this missing dependency should enable the full text support (which I'm assuming was currently disabled in your tests?)

Reply:
Updated.